README.md

```diff
@@ -87,6 +87,7 @@ Please note that this package is still under active development and the API is s
 | `connectToken`      | `string` | [Connect Token](https://docs.arcadia.com/reference/createconnecttoken) |         | Yes      | none    |
 | `newCredentialData` | `object` | Data used to prefill new credentials - only available in 'create' mode          |         | No       | none    |
 | `callbacks`         | `object` | Callback functions                                                              |         | No       | none    |
+| `variant`           | `string` | Set to 'tariff' if using [tariff selection mode](https://docs.arcadia.com/docs/connect-tariff-selection-mode)  |'tariff' or 'connect'        | No       | none    |
```

Comment:
@KristenWAlthoff the question that popped in my head from reading this is if the `'connect'` variant is the same as passing no argument (`None`). If so, should the default be `'connect'`?